src/xeus_sqlite_interpreter.cpp

```diff
@@ -288,7 +306,10 @@ void interpreter::parse_code(const std::vector<std::string>& tokenized_code)
     }
     else
     {
-        publish_stream("stdout", "Load a database to run this command.\n");
+        std::string error = "Load a database to run this command.";
+        m_traceback.push_back("Error: " + error);
+        publish_execution_error("Error", error, m_traceback);
+        m_traceback.clear();
```

Comment:
Maybe raise an exception here instead? It will be catched and published in the `execute_request_impl` by your code.

Reply:
Thanks for the help here :)